projects/plugins/migration/wpcom-migration.php

```diff
@@ -33,25 +33,25 @@
 	exit;
 }
 
-define( 'JETPACK_MIGRATION_DIR', plugin_dir_path( __FILE__ ) );
-define( 'JETPACK_MIGRATION_ROOT_FILE', __FILE__ );
-define( 'JETPACK_MIGRATION_ROOT_FILE_RELATIVE_PATH', plugin_basename( __FILE__ ) );
-define( 'JETPACK_MIGRATION_SLUG', 'jetpack-migration' );
-define( 'JETPACK_MIGRATION_NAME', 'Jetpack Migration' );
-define( 'JETPACK_MIGRATION_URI', 'https://jetpack.com/jetpack-migration' );
-define( 'JETPACK_MIGRATION_FOLDER', dirname( plugin_basename( __FILE__ ) ) );
+define( 'WPCOM_MIGRATION_DIR', plugin_dir_path( __FILE__ ) );
+define( 'WPCOM_MIGRATION_ROOT_FILE', __FILE__ );
+define( 'WPCOM_MIGRATION_ROOT_FILE_RELATIVE_PATH', plugin_basename( __FILE__ ) );
+define( 'WPCOM_MIGRATION_SLUG', 'wpcom-migration' );
+define( 'WPCOM_MIGRATION_NAME', 'Move to WordPress.com' );
+define( 'WPCOM_MIGRATION_URI', 'https://jetpack.com/wpcom-migration' );
```

Comment:
Let's redefine this as https://wordpress.com/

Reply:
Updated, thanks!